source/docs/3/solidwaste_mapping.rst

```diff
@@ -0,0 +1,252 @@
+      :align: center
+
+16. Next select the ``waste_by_station_september_2023`` layer and select :guilabel:`Simple Marker` symbol. Click the drop-down for :guilabel:`Symbol layer type`.
+
+   .. image:: /static/3/solidwaste_mapping/images/16.png
+      :align: center
+	  
+17. Select ``Filled Marker`` as the :guilabel:`Symbol layer type`. We will now change the size of the symbol proportional to the amount of waste collected at the site. To do this, we must apply a Data-defined Override - which can apply a field value or expression to calculate the size for each feature. Click the :guilabel:`Date-defined Override` button next to :guilabel:`Size` and select :guilabel:`Edit`.
+
+   .. image:: /static/3/solidwaste_mapping/images/17.png
+      :align: center
+	  
+18. Enter the follow expression. We are trying to scale the waste amount to a number will be the diameter of the circle on the map. Click :guilabel:`OK`.
+``"Sept2023" / 3``
+
+   .. image:: /static/3/solidwaste_mapping/images/18.png
+      :align: center
+	  
+19. Select the units as :guilabel:`Map Units` (which are in meters) from the drop-down. You will see the circles of different size for each point. The data-defined override button will turn yellow indicating that an override is applied for that value.
+
+   .. image:: /static/3/solidwaste_mapping/images/19.png
+      :align: center
+	  
+20. Let’s explore more advanced styling options. Change the Symbol layer type to :guilabel:`Shapeburst Fill`. Select 2 colors of your choice to render the circles with a gradient fill.
+
+   .. image:: /static/3/solidwaste_mapping/images/20.png
+      :align: center
+	  
+21.Next we will apply a :guilabel:`Drop-shadow effect` to the circles to make them pop-out on the map. These are known as ``Live Layer Effects``. Scroll down and expand the :guilabel:`Layer Rendering` section. Check the :guilabel:`Draw effects` button and click the star button.
+
+   .. image:: /static/3/solidwaste_mapping/images/21.png
+      :align: center
+	  
+22. Enable the :guilabel:`Drop Shadow` option.
+
+   .. image:: /static/3/solidwaste_mapping/images/22.png
+      :align: center
+	  
+23. The map looks pretty good now, but the reader needs to know what values these symbols represent. It will be good to have an interpretable legend. Click :guilabel:`Back` button till you are back in the main Layer Styling dialog. Select :guilabel:`Marker` and click on the :guilabel:`Advanced` button at the bottom. Select :guilabel:`Date-defined Size Legend`.
+
+   .. image:: /static/3/solidwaste_mapping/images/23.png
+      :align: center
+	  
+24. Enter ``Waste Collected (Tonnes)`` as the :guilabel:`Title` and click the :guilabel:`+` button to add legend entries. Since our symbols are scaled by a factor of 3, enter the appropriate value and Label. You will see a nice legend now appear in the Layers panel. The same legend will be available in the ``Print Layout`` if you wished to create a map from this data.
```

Comment:
Expand the 'water by station' layer to show the legend that was created in the Layers panel.

Reply:
Done